docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -46,6 +48,8 @@ The `navigation` prop in the [AppProvider](https://mui.com/toolpad/core/react-ap
 
 The flexibility in composing and ordering these different elements allows for a great variety of navigation structures to fit your use case.
 
+Optionally, a specific `navigation` can also be passed as a prop to the `DashboardLayout` component itself.
```

Comment:
I think this might be better placed as an `:::info` callout so that it isn't missed; and perhaps the demo can explain the usage of this prop

Reply:
I feel like there's no need to demo these besides the callout as it's not the recommended usage so don't want to cause confusion, and the usage is the exact same for in the `AppProvider` so there's nothing new to really exemplify, plus no good place for an extra demo unless we made a whole section about it...

We could make a whole section about it but also thinking it might be a bit overkill, as the sections where we have this information are already the places where readers would look for them.

Reply:
Fair enough, your point about this not being recommended usage and only an additional option makes sense 👍🏻